### PR TITLE
Bd/new slideshow tweaks

### PR DESCRIPTION
**Pull request by bduncan137**

### CODE REVIEW

File: castle/cms/browser/content/slideshow.py [hunk 1]
```diff
@@ -90,8 +92,19 @@ def get_resource_view_url(self, resource):
             url += '/view'
         return url
 
+    def get_domain(self):
+        portal_url = getToolByName(self.context, "portal_url")
+        portal = portal_url.getPortalObject()
+        full_url = portal.absolute_url()
```
Comment (lucid-0) [inline castle/cms/browser/content/slideshow.py:98]: just a note could use api.portal.get().absolute_url()

File: castle/cms/profiles/default/registry/resources.xml [hunk 2]
```diff
@@ -223,4 +223,9 @@
     </value>
   </record>
 
+  <records prefix="plone.resources/slideshow-js"
```
Comment (lucid-0) [inline castle/cms/profiles/default/registry/resources.xml:226]: We'll need to add this to a new upgrade step too so we don't have to re-run default profile on existing sites

File: castle/cms/browser/content/slideshow.py (hunk 1 above)
Comment (bduncan137) [reply castle/cms/browser/content/slideshow.py:98]: that sounds better

File: castle/cms/profiles/default/registry/resources.xml (hunk 2 above)
Comment (bduncan137) [reply castle/cms/profiles/default/registry/resources.xml:226]: k, i'll do that now

**Pull request merged**